core/ExceptionHandler.js

```diff
@@ -62,7 +69,15 @@ export class ExceptionHandler {
 
         this.logException(exception, options);
         if (options.showAlert) {
-            XH.appContainerModel.exceptionDialogModel.show(exception, options);
+            if (!options.showWithToast) {
+                XH.toast({
+                    message: exception.message,
+                    intent: 'danger',
+                    timeout: 10000
```

Comment:
+ Can we provide a link in the toast to bring up the full dialog with the show call on line 79?
+ struggling with whether we need to customize this toast?  Seems like we could do it with a global setting on ExceptionHandler in a future release if we ever needed to do that.  
+ Seems like we need a bundle of options we can set on ExceptionHandler globally when the app is loading ("ExceptionHandlerOptions"), this could include, say, the REDACT text, but also whether toast or dialog is the default, toast defaults, etc. etc.